lib/rucio/core/did_meta_plugins/filter_engine.py

```diff
@@ -255,15 +255,72 @@ def _translate_filters(self, model_class, strict_coerce=True):
         self._filters = filters_translated
         return list(mandatory_model_attributes)
 
+    def create_mongo_query(self, additional_filters={}):
```

Comment:
created new function to construct mongo queries (analogous to create_sqla_query function)